core/src/generate_types.rs

```diff
@@ -506,11 +512,14 @@ impl TypePathType {
                 syn::Type::Path(path)
             }
             TypeDef::Phantom(_) => {
+                /* TODO: [now]: As soon as branch `aj-substrate` is updated to contain new code for
+                 * TypeDefPhantom this goes back, use `()` for the time being.
                 let type_param = params
                     .iter()
                     .next()
                     .expect("a phantom type should have a single type parameter");
-                let type_path = syn::parse_quote! { core::marker::PhantomData<#type_param> };
+                */
```

Comment:
Yes, `aj-substrate` is in flux atm - should stabilize in the next day or two once I get `polkadot` compiling.